index.html

```diff
@@ -1,60 +1,260 @@
 <!DOCTYPE html>
-<html lang="en-ca">
+<html lang="en">
   <head>
-    <!-- Set the page title -->
-    <title>Assignment 01 - To Be Completed</title>
-
-    <!-- Set the charset we're using to UTF-8 -->
     <meta charset="UTF-8" />
-
-    <!-- Set the screens default zoom / scale to 1 -->
+    <title>Assignment 01</title>
     <meta name="viewport" content="width=device-width, initial-scale=1" />
-
-    <!-- Specifies the document mode for Internet Explorer -->
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
 
     <!-- CSS -->
     <!-- The following CSS has been added so that the page
     doesn't look super awful -->
     <!-- Ignore everything in the <style> tag for now -->
     <!-- We will learn about CSS later in the course -->
+    <!-- Scroll down for instructions on how to complete the tutorial -->
     <style>
-      /*
-        These styles set the font, font-size, and add a
-        little bit of space between lines
-      */
       body {
         font-family: Arial, Helvetica, sans-serif;
         font-size: 1.2rem;
         line-height: 1.6;
       }
-      /*
-        These styles make sure that any super wide images
-        don't overflow the page width
-      */
       img {
         width: 100%;
         display: block;
         margin: 0 auto;
       }
+      body {
+        background-color: rgb(1, 1, 12);
+        font-family: Arial, Helvetica, sans-serif;
+        font-size: 1.2rem;
+        line-height: 1.6;
+        color: #fefffee7;
+      }
     </style>
   </head>
   <body>
-    <!--
+    <!-- INSTRUCTIONS -->
+    <!-- TO DO: Add markup to the following content so that it has the proper content structure -->
+    <!-- You will need to use the following tags: h1, h2, h3, p, ul, li, a, section, nav, header, footer, main, input, label, fieldset, select, form, table, thead, tobody, tfoot, tr, td, img  -->
+    <!-- The correct element appears ABOVE the text -->
+    <!-- You may remove the extra comments when you finish, but you don't need to -->
+
+    <header>
+      <h2>Mercedes-Benz</h2>
+
+      <p>The Luxury Of Mercedes</p>
+
+      <img
+        src="./images/il_fullxfull.3924045420_tja0.avif"
+        alt="Mercedes Benz Logo"
+      />
+
+      <nav>
+        <h3>Menu</h3>
+        <ul>
+          <li><a href="https://www.mercedes-benz.ca/en/all-vehicles">Car Models</a></li>
+          <li><a href="https://www.mercedes-benz.ca/en/dealers">Dealerships</a></li>
+          <li><a href="https://www.mercedes-benz.ca/en/shop-mercedes-benz-collection">Accessories</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <img
+        src="./images/the-mercedes-benz-eqs-manufaktur-signature-edition.jpg"
+        alt="Latest model of Mercedes-Benz"
+      />
+
```

Comment:
This `img` is orphaned and needs to belong to either the header or footer or main